src/main/scss/templates.base/_li-nav-page.scss

```diff
@@ -17,7 +16,7 @@
 		padding-left: $size-base * 4;
```

Comment:
Can we make it $size-base * 3 (or $size-base * 2.5) for the padding-left? It looks a little too spacious on my screen.

![image](https://cloud.githubusercontent.com/assets/13812345/26492245/02a0dd82-41c8-11e7-9033-ad7ad552fc07.png)